internal/core/processing.go

```diff
@@ -192,7 +192,7 @@ func ProcessBookmark(req ProcessRequest) (book model.Bookmark, isFatalErr bool,
 	return book, false, nil
 }
 
-func DownloadBookImage(url, dstPath string) error {
+func DownloadBookImage(fs afero.Fs, url, dstPath string) error {
```

Comment:
should we use `fs` instrad of `os` in this function too?
`tmpFile, err := os.CreateTemp("", "image")` or `os.Remove(tmpFile.Name())`

Reply:
Yeah, what I'm doing here is creating the temporary file locally and then copying it to the destination storage using the storage domain.